python/sparkdl/graph/utils.py

```diff
@@ -72,12 +72,13 @@ def get_op(tfobj_or_name, graph):
         raise TypeError('invalid op request for [type {}] {}'.format(type(name), name))
     _op_name = op_name(name, graph=None)
     op = graph.get_operation_by_name(_op_name)
-    assert isinstance(op, tf.Operation), 'expect tf.Operation, but got {}'.format(type(op))
+    err_msg = 'cannot locate op {} in the current graph, got [type {}] {}'
+    assert isinstance(op, tf.Operation), err_msg.format(_op_name, type(op), op)
```

Comment:
Changed error message to better align to the intended usage of the function rather than focusing locally on this assertion statement.